boltdesignsystem/bolt
# COM-707 Raise signifier link above headline link

## Jira

https://pegadigitalit.atlassian.net/browse/COM-707

## Summary

Updates the Listing Teaser to allow signifier link to have a higher stacking order than the main headline link.

## Details

1. HTML in twig template is modified to achieve stacking order. All elements that might need a higher stacking order will need to come after the headline.
2. CSS is adjusted to account for possible types of action element used for a clickable signifier.

## How to test

Run the branch locally and view docs under Listing Teaser. In the Search Results demo page, go to the Related Videos section, the video thumbnail should open a video modal, the headline link should lead to a page.
